src/tuplet.ts

```diff
@@ -273,6 +274,8 @@ export class Tuplet extends Element {
       // y_pos = first_note.getStemExtents().topY - 10;
 
       for (let i = 0; i < this.notes.length; ++i) {
+        // Skip Ghostnotes
+        if (this.notes[i] instanceof GhostNote) continue;
```

Comment:
Here too, it feels like way too much of an intrusion to explicitly check for GhostNote. I'd rather it break than fail silently.

Reply:
I am allowing gohstnotes in the middle of beams and tuplets. Should that fail?

Reply:
Changed to `hasStem()`